src/main.cpp

```diff
@@ -992,6 +994,153 @@ bool CheckTransaction(const CTransaction& tx, CValidationState &state)
     return true;
 }
 
+extern secp256k1_context* secp256k1_bitcoin_verify_context;
```

Comment:
Nit: would like this in consensus/* rather than in main.cpp despite depneding on CCoinsViewCache/coins.